flood_adapt/object_model/hazard/hazard.py

```diff
@@ -883,6 +881,12 @@ def calculate_rp_floodmaps(self):
 
         # 1a: make a table of all water levels and associated frequencies
         zs = xr.concat(zs_maps, pd.Index(frequencies, name="frequency"))
+        # Get the indices of columns with all NaN values
+        nan_cells = np.where(np.all(np.isnan(zs), axis=0))[0]
+        # fill nan values with minimum bed levels in each grid cell, np.interp cannot ignore nan values
+        for i in range(zs.shape[0]):
+            zs[i, :] = xr.where(np.isnan(zs[i, :]), zb, zs[i, :])
```

Comment:
Not sure about how to call np.repmat exactly, you need to try
```suggestion
        zs = xr.where(np.isnan(zs), np.repmat(zb,zs.shape[0],1), zs)
```